examples/wordpress/README.md

```diff
@@ -0,0 +1,153 @@
+# How To Use NFS Persistent Volumes
+
+The purpose of this guide is to help you understand storage provisioning with NFS by creating a WordPress blog and MySQL database.
+Both the blog and database require persistent storage.  
+
+This guide assumes knowledge of OpenShift fundamentals and that you have a cluster up and running.
+
+## NFS Provisioning
+
+We'll be creating NFS exports on the local machine.  The provisioning process will be similar regardless of where the NFS server lives.
```

Comment:
I added that these come from Fedora, which should translate seamlessly to centos/rhel, correct?  I added verbiage that says it might be different on other distros or NFS servers, too.